### ISSUE TITLE
RWD error: request entity too large

### ISSUE THREAD

**Issue by ajrobbins** [WikiWatershed/model-my-watershed#1176]
When trying to delineate a watershed on staging within ITSI, I got an error in Analyze with the following in dev tools: `413 Request Entity Too Large nginx/1.8.1`

I've so far been unable to replicate it in or outside of ITSI.
![screen shot 2016-03-10 at 11 08 24 am](https://cloud.githubusercontent.com/assets/12868342/13675663/ad11a482-e6b0-11e5-9039-34cd5c768ac6.png)

**Comment by ajrobbins** [comment]
Was able to replicate on staging outside of ITSI using this click point: `[39.987642831840844, -75.19819736480713]`

**Comment by mmcfarland** [comment]
At a glance, this appears to be that the generated watershed, which is posted to the analyze endpoint, is too large.  We have previously increased the max size, but that's a little risky and temporary.  We may need to increase again temporarily to allow complex watersheds to be posted while we come up with an alternative approach.

**Comment by hectcastro** [comment]
Looking at the Nginx configuration, we have this at 5MB. The generated watersheds are that big?

If the request is being generated by us, maybe we can take a look at making sure headers are being set to enable compression. That might help reduce these payloads a significant amount.

**Comment by ajrobbins** [comment]
The watershed that RWD generated here appears to be the Schuylkill HUC-8 (it's shaped like a rabbit). Shouldn't the analyze endpoint be able to handle that, since it's pretty much the same as a select by boundary? Unless it's different coming from RWD?

**Comment by kdeloach** [comment]
Running RWD locally, I get this watershed for coordinates `39.987642831840844, -75.19819736480713`. The file size is about 3kb.

![image](https://cloud.githubusercontent.com/assets/43062/13678528/3437ac52-e6bd-11e5-9f95-2e72bc5d050f.png)

**Comment by mmcfarland** [comment]
@hectcastro We are generating the request, but I'm not aware of headers on the client request that will let the (presumably) browser gzip on request.  Is that a thing you're aware of?  It'd be great.

@kdeloach strange that your watershed is so different (and simplified).  Not sure what to make of that yet.

**Comment by kdeloach** [comment]
I'm not sure why the lat/lng I tested resulted in such a small watershed. Although, I was able to generate both small and large watersheds in the general vicinity of that test island.

There seems to be a large variance in the generated watersheds even for points that are very near to each together.

Here is an example of a large watershed that I generated. The analyze request body Content-Length was 1248573 (1.2MB). Analyze took about 40 seconds to complete.

![image](https://cloud.githubusercontent.com/assets/43062/13680475/ac839d02-e6c6-11e5-86bd-8215ee018d5b.png)
